src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +381,106 @@ public InputStream body() throws IOException {
         }
     }
 
+    /**
+     * Fake decorator.
+     * @since 0.16
+     */
+    final class Fake implements RqMultipart {
+
+        /**
+         * Fake boundary constant.
+         */
+        private static final String BOUNDARY = "AaB02x";
+        /**
+         * Fake multipart request.
+         */
+        private final RqMultipart fake;
+
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         *  @todo #252:30min/DEV New ctor should be created and expect
+         *  instances of Request as dispositions. Dispositions should be
+         *  multipart request body holder. New ctor should call
+         *  {@code RqMultipart.Base}. {@code Fake.fakebody} method should be
+         *  refactored and Fake(final Request req, final String... dispositions)
+         *  ctor should call new constructor using dispositions like
+         *  {@code
+         *  this(
+         *  req,
+         *  new RqWithHeader(
+         *  req,
+         *  // @checkstyle LineLength (1 line)
+         *  new RqLive(req.head().iterator().next() + "\r\n" + "Content-Disposition", "form-data; name=\"address\"")
+         *  ),
+         *  new RqWithHeader(
+         *  req,
+         *  // @checkstyle LineLength (1 line)
+         *  new RqLive(req.head().iterator().next() + "\r\n" + "Content-Disposition", "form-data; name=\"file\"")
+         *  )
+         *  );
+         *  }
+         */
+        public Fake(final Request req, final String... dispositions)
+            throws IOException {
+            this.fake = new RqMultipart.Base(
+                new RqSimple(
+                    new RqWithHeader(
+                        req,
+                        // @checkstyle LineLength (2 line)
+                        // @checkstyle MultipleStringLiteralsCheck (1 line)
+                        "Content-Type", String.format("multipart/form-data; boundary=%s", BOUNDARY)
```

Comment:
@erimerturk you can reformat this code to make it fit into 80 characters. just put `String.format()` to the next line